src/Microsoft.VisualStudio.Threading/IAsyncDisposable.cs

```diff
@@ -11,6 +11,9 @@ namespace Microsoft.VisualStudio.Threading
     /// <summary>
     /// Defines an asynchronous method to release allocated resources.
     /// </summary>
+    /// <remarks>
+    /// Consider implementing <see cref="System.IAsyncDisposable"/> instead.
```

Comment:
💡 Since a non-optional dependency on `Microsoft.Bcl.AsyncInterfaces` has been added, consider marking this interface obsolete

Reply:
I did consider it. But that would make everyone already implementing the interface suddenly get compiler warnings/errors. Some of these folks would "fix" it by *removing* the interface from the class declaration, which would be a breaking change although they may not realize it. 
I don't want to encourage that bad behavior, so I went with documenting it as a suggestion.

Reply:
Perhaps an analyzer? Would be cheap to implement.

Reply:
And what would it call out? Analyzers aren't in a good position to block *new* uses of an API. It would call out all of them, and my point is that *all existing users* should continue being users. Folks who implemented it should continue to do so, and folks who support this interface by type checking it and calling `DisposeAsync` if it's implemented should continue to do so. 
What would you have the analyzer call out?

Reply:
> What would you have the analyzer call out?

Code that implements `Microsoft.VisualStudio.Threading.IAsyncDisposable` should also have `System.IAsyncDisposable` listed as an implemented interface.

Reply:
Cool. I've filed an issue to track that suggestion.